src/object/object_representation.c

```diff
@@ -743,10 +671,10 @@ or_mvcc_set_insid (OR_BUF * buf, MVCC_REC_HEADER * mvcc_rec_header)
  * mvcc_falgs(in)  : MVCC flags
  * error(out): NO_ERROR or error code
  */
-STATIC_INLINE union DELID_CHN
-or_mvcc_get_delid_chn (OR_BUF * buf, int mvcc_flags, int *error)
+STATIC_INLINE MVCCID
+or_mvcc_get_delid (OR_BUF * buf, int mvcc_flags, int *error)
```

Comment:
Update the function description and apply the same format to `or_mvcc_get_chn` and `or_mvcc_set_chn`.